Topics/Tech_Stacks/React.md

```diff
@@ -1,37 +1,62 @@
-https://reactjs.org/tutorial/tutorial.html
+[React Documentation](https://react.dev/)
+The official React documentation is a great way to get started with the library, but also indispensible to have bookmarked for reference.
 
+[Coursera Learn React Basics](https://www.coursera.org/learn/react-basics)
+A free course provided by Coursera that covers the basics of React, including State, navigation, and taking user input via forms.
 
-React tutorial by MDN Web Docs. Useful readings for if you have time, but not necessary by any means.\
-https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Client-side_JavaScript_frameworks/React_getting_started
+[Free Scrimba React Course](https://scrimba.com/learn/learnreact)
+A free course provided by Scrimba that teaches React fundamentals through the creation of various applications including an AirBnB clone, a meme generator, and a notes app.
 
+[Official React Tutorial](https://reactjs.org/tutorial/tutorial.html)
+Basic React tutorial provided by the official React website. This tutorial will guide you through a very basic tutorial that you can do straight from your browser. 
 
-Javascript frameworks tutorial provided by Microsoft. Useful readings, but not necessary to learn React. \
-https://learn.microsoft.com/en-us/windows/dev-environment/javascript/
+[React JS Crash Course](https://www.youtube.com/watch?v=w7ejDZ8SWv8)
+[One Hour React Tutorial](https://www.youtube.com/watch?v=b9eMGE7QtTk)
+One hour quickstart tutorials about React. These YouTube crash course provides a great overview of React and various important components that you will be using often. Very useful if you are learning React under a strict time limit for whatever reason.
 
+[Front-End Mentor](https://www.frontendmentor.io/)
+These frontend challenges for HTML, CSS and JavaScript will help you review core skills while also making interesting projects.
 
-1 hour in depth tutorials about React. These Youtube crash course provides a great in depth look at React and the various important components that you will be using most often. Very useful if you are learning React under a strict time limit for whatever reason. \
-https://www.youtube.com/watch?v=w7ejDZ8SWv8 \
-https://www.youtube.com/watch?v=b9eMGE7QtTk 
+[Mozilla React Tutorial](https://developer.mozilla.org/en-US/docs/Learn/Tools_and_testing/Client-side_JavaScript_frameworks/React_getting_started)
+React tutorial by MDN Web Docs. Useful readings for if you have time.
 
+[Microsoft JavaScript Tutorial](https://learn.microsoft.com/en-us/windows/dev-environment/javascript/)
+JavaScript frameworks tutorial provided by Microsoft. Useful readings, but not necessary to learn React. 
 
-Online development environment to try React out in (Among other frontend tools). Great for testing the waters with React to see if it will be something you want to learn or not. \
-https://codepen.io/
+### Useful React Tools and Libraries
 
+[Create React App](https://create-react-app.dev/)
```

Comment:
Create React App is criticized for being slow, in terms of the actual react website you make being slow. A substitute would be Vite, which works very similar to create react app. Consider putting this in.

I don't know if Next.js should be mentioned or is already mentioned here.